swift/example-qr-generator/QrUITests/QrUITests.swift

```diff
@@ -0,0 +1,41 @@
+//
```

Comment:
I assume this tests are for sample app. Its sample so no need to have tests. Can we remove it?

Reply:
Done